test-packages/test-services-e2e/v4/test-service/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2019",
+    "target": "es2021",
```

Comment:
[q] If I look at the [specs](https://www.typescriptlang.org/tsconfig#target) there are `es2022`and `esnext` as even newer standards. Is there a reason why we use 2021?

Reply:
With es2022, it introduces lots of compile time/runtime changes and the clients are generated differently.
I switched back to es2021, and now everything are solved right away.
Therefore, I would make this as a first step and merge it.
Based on this, I'll continue my investigation.